include/deal.II/matrix_free/tensor_product_kernels.h

```diff
@@ -1751,7 +1952,19 @@ namespace internal
     static void
     apply(const Number2 *DEAL_II_RESTRICT shape_data,
```

Comment:
... the "old" apply function is static, which does not work for the non-templated case, where we pass in the bounds via the constructor.